src/assisted_test_infra/test_infra/helper_classes/cluster.py

```diff
@@ -317,8 +317,8 @@ def set_network_params(self, controller=None):
         else:
             log.info("Assigning VIPs statically")
             access_vips = controller.get_ingress_and_api_vips()
-            api_vip = access_vips["api_vip"]
-            ingress_vip = access_vips["ingress_vip"]
+            api_vip = access_vips["api_vip"] if access_vips else None
+            ingress_vip = access_vips["ingress_vip"] if access_vips else None
```

Comment:
Why getting api vip and ingress vip as env variables and not using them?

Reply:
You might want to return those value from `get_ingress_and_api_vips` (Or that I'm missing something)?

Reply:
Underlying platform is managed by the controller. While the controller can provide the VIP's IPs, if they are provided as env variables that are unrelated to the underline platform, the controller should not return them.
Instead of duplicating everything at the controller level, we should try to implement it in the most generic way possible. 
We have 4 major cases here:
1. DHCP
2. Static 
3. Taking free IP
4. Custom - by controller
All controllers should share the first three cases

Reply:
the logic here is: If the controller provides vips then use them else take it from the config (env variable)

Reply:
I used env variables because sometimes the VIPs allocation has different logic between Envs like CI and Local and cannot be determine by the controller especially for real virtualization solutions

Reply:
> I used env variables because sometimes the VIPs allocation has different logic between Envs like CI and Local and cannot be determine by the controller especially for real virtualization solutions

But you are not using those in your code. You added api and ingress vips to the variables defaults but they are not being used elsewhere in the code

Reply:
please see:
https://github.com/openshift/assisted-test-infra/blob/8fdf16a0b255f2104de00f254f9bbb4ed8fbaaea/src/assisted_test_infra/test_infra/helper_classes/cluster.py#L406